src/aws/osml/model_runner/database/region_request_table.py

```diff
@@ -152,3 +205,32 @@ def get_region_request(self, region_id: str, image_id: str) -> Optional[RegionRe
         except Exception as err:
             logger.warning(GetRegionRequestItemException(f"Failed to get RegionRequestItem! {err}"))
             return None
+
+    def add_tile(self, image_id: str, region_id: str, tile: ImageRegion, state: str) -> dict[str, Any]:
+        """
+        Append tile to the 'succeeded_tiles' property of a DDB item in the table.
+
+        :param image_id: str = the id of the image request we want to update
+        :param region_id: str = the id of the region request we want to update
+        :param tile: ImageRegion = list of values to append to the 'succeeded_tiles' property
+        :param state: str = state of the tile to add, i.e. succeeded or failed
```

Comment:
Is this state parameter actually used somewhere in this function?

Reply:
Fixed